src/applications/check-in/components/pages/AppointmentDetails/index.jsx

```diff
@@ -105,6 +118,12 @@ const AppointmentDetails = props => {
     );
   }
 
+  const showReviewButton =
+    isPreCheckIn &&
+    !isUpcoming &&
+    pages.length > 4 &&
```

Comment:
no need to change, but I wonder if we even need this check, since complete will have been set before they get a chance to see this when there is less than 4 pages.

Reply:
I had that thought too but I was thinking `!getPreCheckinComplete(window)?.complete` resolves to true before you answer the questions. Does the session value get set with the initial payload after login? If it does then yeah we probably don't need it and I can remove one of the unit tests.